docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -42,7 +42,31 @@ upgrading a cluster involves leaving the old cluster running on the old version,
 cluster on the new version, porting across your database schema, and finally thinking about your data.
 Read on for details.
 
-### Port the database schema
+
+### In-Place upgrade
+
+#### Steps
+
+1. Download the [new ksqlDB version](https://docs.confluent.io/platform/current/installation/installing_cp/zip-tar.html#get-the-software) to upgrade to.
+2. Set it up with all required configs. For the vast majority of upgrades, you may simply copy existing configs over to the new version.
+3. Copy source UDFs and recompile against the new ksqlDB version, to accommodate any changes to the UDF framework.
+4. Stop ksqlDB process of current version and start newer version with latest ksqlDB config file which you created in step 2.
+5. KsqlDB will start executing queries and rebuild all states using command topic because the value of "ksql.service.id" is the same.
```

Comment:
```suggestion
5. ksqlDB starts executing queries and rebuilds all states, using the command topic, because the value of `ksql.service.id` is the same as the previous version.
```